db/migrate/20160924213020_change_katello_widget_names.rb

```diff
@@ -0,0 +1,23 @@
+class ChangeKatelloWidgetNames < ActiveRecord::Migration
+  def up
+    Widget.where(:name => 'Errata Widget').update_all(:name => 'Latest Errata')
```

Comment:
The `Widget` class will need to be redefined in this migration to avoid future changes. An example can be seen here:

https://github.com/Katello/katello/blob/master/db/migrate/20160222143432_move_system_description_to_host.rb#L2-L4